### PR TITLE
NN-3686: Added hold-against-transfer alert details to scheduled moves page

### PR DESCRIPTION
**Pull request by sp-watson**

### CODE REVIEW

File: backend/controllers/whereabouts/scheduledMoves.ts [hunk 1]
```diff
@@ -58,6 +72,39 @@ export default ({ prisonApi, offenderSearchApi }) => {
...
+    const activeHoldAgainstTransferAlertDetails = await prisonApi.getAlertsForLatestBooking(context, {
+      offenderNo: prisonerDetails.prisonerNumber,
+      alertCodes: relevantAlertsForHoldAgainstTransfer,
+      sortBy: 'dateCreated',
+      sortDirection: 'DESC',
+    })
+
+    return {
+      prisonerNumber: prisonerDetails.prisonerNumber,
+      displayId: uuidv4(),
```
Comment (sp-watson) [inline backend/controllers/whereabouts/scheduledMoves.ts:97]: I tried to do this in the NJK, but couldn't get it working. I have found an npm package that might do the trick...

File: integration-tests/integration/whereabouts/scheduledMoves.spec.js [hunk 2]
```diff
@@ -326,6 +379,7 @@ context('Scheduled movements', () => {
             expect(rows[0].property).contains('Valuables - Box 14')
             expect(rows[0].property).contains('Confiscated - Box 15')
             expect(rows[0].alerts).contains('ACCT open')
+            expect(rows[0].alerts).contains('Hold against transfer')
```
Comment (sp-watson) [inline integration-tests/integration/whereabouts/scheduledMoves.spec.js:382]: I am in the process of adding a test for the dialog itself

File: integration-tests/plugins/index.js [hunk 3]
```diff
@@ -154,6 +154,7 @@ module.exports = (on) => {
 
     stubAlertTypes: () => Promise.all([prisonApi.stubAlertTypes()]),
     stubAlertsForBooking: (alerts) => Promise.all([prisonApi.stubAlertsForBooking(alerts)]),
+    stubAlertsForLatestBooking: (alerts) => Promise.all([prisonApi.stubAlertsForLatestBooking(alerts)]),
```
Comment (JamieDraperUK) [inline integration-tests/plugins/index.js:157]: Why a Promise.all here?

File: static/js/main.js [hunk 4]
```diff
@@ -18,12 +18,12 @@ $(document).ready(function() {
...
 
-  $('.js-prisoner-search-clear-alerts').click(function(e) {
+  $('.js-prisoner-search-clear-alerts').click(function (e) {
     e.preventDefault()
     $('.js-prisoner-search-alerts input[type=checkbox]').prop('checked', false)
   })
 
-  $('#notification_dismiss_box').click(function() {
+  $('#notification_dismiss_box').click(function () {
     let $notification = $('#notification-bar')
```
Comment (JamieDraperUK) [inline static/js/main.js:27]: Can you use const here?

**Review by JamieDraperUK — DISMISSED** [review]
LGTM

**Pull request merged**